packages/@lwc/perf-benchmarks-components/rollup.config.mjs

```diff
@@ -5,11 +5,14 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 
-import path from 'path';
+import { fileURLToPath } from 'node:url';
+import path from 'node:path';
 import glob from 'glob';
 import lwc from '@lwc/rollup-plugin';
 import replace from '@rollup/plugin-replace';
-import { generateStyledComponents } from './scripts/generate-styled-components';
+import { generateStyledComponents } from './scripts/generate-styled-components.mjs';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
```

Comment:
`__dirname` does not exist in ESM; this works instead.

Reply:
@nolanlawson why is this not a problem in the other rollup config files, like this one https://github.com/salesforce/lwc/blob/master/packages/%40lwc/engine-dom/scripts/rollup.config.js#L84

Reply:
Because these files are in CommonJS format. They use `require` rather than `import`.

Reply:
💡